qiskit/extensions/quantum_initializer/uc.py

```diff
@@ -305,7 +307,17 @@ def ucg(self, gate_list, q_controls, q_target, up_to_diagonal=False):
     if num_contr != len(q_controls):
         raise QiskitError("Number of controlled gates does not correspond to the number of"
                           " control qubits.")
-    return self.append(UCG(gate_list, up_to_diagonal), [q_target] + q_controls)
+    return self.append(UCGate(gate_list, up_to_diagonal), [q_target] + q_controls)
 
 
-QuantumCircuit.ucg = ucg
+def ucg(self, angle_list, q_controls, q_target, up_to_diagonal=False):
+    """
+    Deprecated version of uc.
+    """
+    import warnings
+    warnings.warn('qc.ucg is deprecated, use qc.uc instead!', DeprecationWarning, 2)
```

Comment:
This message does not match the style of other deprecation messages.
```suggestion
    warnings.warn('The QuantumCircuit.ucg() method is deprecated as of 0.14.0, and '
                  'will be removed no earlier than 3 months after that release date. '
                  'You should use the QuantumCircuit.uc() method instead.',
                  DeprecationWarning, stacklevel=2)
```